config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  namespace :admin do
+    resources :articles
+    resources :users
+
+    root to: "articles#index"
```

Comment:
I'm not sure we need to redefine the `root` :thinking:

Reply:
This specific `root` is for the `admin user` only.

When the `admin user` logs in, for now, they are routed to the index of all the articles in the app.

Reply:
Cool cool
it makes sense :+1: